superset-frontend/src/datasource/DatasourceModal.tsx

```diff
@@ -51,6 +51,11 @@ const StyledDatasourceModal = styled(Modal)`
   }
 `;
 
+const StyledAlert = styled(Alert)`
+  margin-top: ${({ theme }) => theme.gridUnit * 4}px;
+  margin-bottom: ${({ theme }) => theme.gridUnit * 4}px;
```

Comment:
Again, I'm wondering if this is something that should be part of the Alert itself (as a classname, or as a prop (which can be used by Emotion)). Also, this one is 4 gridUnits bottom margin, where the DatasourceEditor one is 5 gridUnits. Hoping we can strive for consistency here, unless there are special use cases floating around.